### PR TITLE
copy config method

### PR DESCRIPTION
**Pull request by amazingalek**

### CODE REVIEW

File: OWML.ModLoader/ModData.cs [hunk 1]
```diff
@@ -22,14 +22,14 @@ public class ModData : IModData
                                  Config != null && Config.RequireVR ||
                                  Config == null && DefaultConfig != null && DefaultConfig.RequireVR;
 
-        private IModConfig _configSnapshot;
+        private readonly IModConfig _configSnapshot;
```
Comment (TAImatem) [inline OWML.ModLoader/ModData.cs:25]: won't really work if I would use Config.Copy in UpdatedSnapshot

Comment (amazingalek) [reply OWML.ModLoader/ModData.cs]: ok, so like this? (pushed)

**Comment by TAImatem** [comment]
Also add ?s just in case :wink:

**Comment by TAImatem** [comment]
Did it myself 🤷‍♂️ https://github.com/amazingalek/owml/pull/262

**Comment by amazingalek** [comment]
@TAImatem wanna merge this?

**Pull request merged**